openmls/src/extensions/mod.rs

```diff
@@ -58,6 +58,10 @@ use tls_codec::{
 #[cfg(test)]
 mod tests;
 
+#[cfg(feature = "extensions-draft-08")]
+/// A ComponentId that uniquely identifies a component within the scope of an application.
+pub type ComponentId = u32;
```

Comment:
This is u16 now. https://messaginglayersecurity.rocks/mls-extensions/draft-ietf-mls-extensions.html#name-component-ids